test/e2e/actuals_test.go

```diff
@@ -103,6 +103,21 @@ func validateOverrideAnnotationOfConfigMapOnCluster(cluster *framework.Cluster,
 	return nil
 }
 
+func validateConfigMapNoAnnotationKeyOnCluster(cluster *framework.Cluster, key string) error {
+	workNamespaceName := fmt.Sprintf(workNamespaceNameTemplate, GinkgoParallelProcess())
+	appConfigMapName := fmt.Sprintf(appConfigMapNameTemplate, GinkgoParallelProcess())
+
+	configMap := &corev1.ConfigMap{}
+	if err := cluster.KubeClient.Get(ctx, types.NamespacedName{Namespace: workNamespaceName, Name: appConfigMapName}, configMap); err != nil {
+		return err
+	}
+
+	if len(configMap.Annotations) == 0 || configMap.Annotations[key] == "" {
```

Comment:
```suggestion
        if _, exist :=  configMap.Annotations[key]; !exist {
```
just in case, annotation value is empty but key is there.